packages/aztec.js/src/note/index.js

```diff
@@ -178,14 +196,26 @@ class Note {
         return `0x${a}${k}${ephemeral}`;
     }
 
+    /**
+     * Grant an Ethereum address access to the viewing key of a note
+     *
+     * @param {Object} access mapping between an Ethereum address and the linked publickey
```

Comment:
Not an object. Or at least not strictly an object.

Reply:
updated